lib/pbench/server/api/resources/users_api.py

```diff
@@ -269,14 +268,22 @@ def post(self):
                         "message": "failure message"
                     }
         """
-        auth_token = self.auth.token_auth.get_auth()["token"]
-        user = self.auth.token_auth.current_user()
+        auth_token = self.auth.get_auth_token(self.logger)
+        user = Auth.verify_auth(auth_token=auth_token)
```

Comment:
This is the call that actually looks up the token in the database to find the user associated with the token?

Reply:
Yes